README.md

```diff
@@ -88,10 +88,17 @@ New for Dockerfile
 
 to build docker image : ( in the same folder : e.g. DevOps-Course-Starter-Moudule-2)
 
-$ docker build --tag todo-app .
+to build development and production, run the following command 
 
-to run it 
-
-$ docker run -d -p 5000:5000 --env-file .env todo-app
+Develpment :
+```bash
+$ docker build --target development --tag todo-app:dev . 
+$ docker run -d -p 5000:5000 --env-file .env todo-app:dev
```

Comment:
It would be nice to update this to use the bind mount to allow for hot reloading, which might look like:
```suggestion
$ docker run -d -p 5000:5000 --env-file .env --mount type=bind,source="$(pwd)"/todo_app,target=/DevOpsEx5/todo_app todo-app:dev
```